extensions/wikia/AdEngine/js/lookup/prebid/adaptersPerformanceTracker.js

```diff
@@ -78,16 +78,26 @@ define('ext.wikia.adEngine.lookup.prebid.adaptersPerformanceTracker', [
 	}
 
 	function getParamsFromBidForTracking(bid) {
```

Comment:
Oh, it's not public :/ We'll test it while testing integration then, I guess ([ADEN-5062](https://wikia-inc.atlassian.net/browse/ADEN-5062)).